src/pages/index.js

```diff
@@ -1,96 +1,178 @@
-// Initialize all my instances
-cardsList.renderItems();
-
-// All the rest
+let cardsList, myId;
+Promise.all([api.getUserInfo(), api.getInitialCards()]).then(
+  ([userData, cardsData]) => {
+    myId = userData._id;
+    newUser.setUserInfo(userData);
+    cardsList = new Section(
+      {
+        data: cardsData,
+        renderer: renderCard,
+      },
+      cardElements
+    );
+    cardsList.renderItems(cardsData);
+  }
+);
 
 /*----------------------------------------------------------------------------*/
 /*                                  Validation                                */
 /*----------------------------------------------------------------------------*/
 
 const editValidator = new FormValidator(validationSettings, profileForm);
 const addValidator = new FormValidator(validationSettings, cardForm);
+// const confirmValidator = new FormValidator(validationSettings, confirmForm);
+const changeValidator = new FormValidator(validationSettings, changeForm);
 editValidator.enableValidation();
 addValidator.enableValidation();
+changeValidator.enableValidation();
 
 /*----------------------------------------------------------------------------*/
 /*                                  Functions                                 */
 /*----------------------------------------------------------------------------*/
 
+async function handleLikeClick(card) {
+  const isLiked = card.cardIsLiked();
+  await api
+    .changeLikeStatus(card._cardId, isLiked)
+    .then((data) => {
+      this.setLikeCounter(data.likes);
+    })
+    .catch((err) => {
+      console.error(err);
+    });
+}
+
+function handleTrashIcon(item) {
+  popupConfirmForm.open(item);
+}
+
 function handleCardClick(item) {
   popupWithImage.open(item);
 }
 
 function renderCard(item) {
-  const card = new Card(item, cardTemplate, handleCardClick);
+  const card = new Card(
+    item,
+    myId,
+    cardTemplate,
+    handleCardClick,
+    handleTrashIcon,
+    handleLikeClick
+  );
   const cardElement = card.getView();
   return cardElement;
 }
 
 /*----------------------------------------------------------------------------*/
-/*                               Event Handlers                              */
+/*                               Event Handlers                               */
 /*----------------------------------------------------------------------------*/
 
-const newUser = new UserInfo({
-  nameSelector: ".profile__title",
-  jobSelector: ".profile__description",
-});
+async function handleConfirmFormSubmit(idCard) {
+  await api.deleteCard(idCard).then(() => {
+    cardsList.removeItem(idCard);
+  });
+}
 
-function handleProfileFormSubmit({ title, description }) {
-  newUser.setUserInfo({ name: title, job: description });
-  popupNameForm.close();
+function handleChangeFormSubmit(items) {
+  api
+    .updateAvatarPicture(items.link)
+    .then((response) => {
+      newUser.setUserInfo(response);
+      popupChangeForm.close();
+      changeForm.reset();
```

Comment:
According to the task you need to have method `close` inside `PopupWithForm` where you need to **reset** the form not to duplicate it in `index.js`